src/components/cesium_map/cesium_UI.js

```diff
@@ -303,13 +303,6 @@ class CesiumMap extends React.Component {
     };
   };
 
-  // function that updates the points of the layer 
-  redrawPointLayer = (mapInfo, newSearchFields) => {
-    searchFields = newSearchFields; 
-    clearBoundingBox(true);
-    this.updatePrimitive(mapInfo.latitude, mapInfo.longitude);
-  }
-
   // This is a initial function in react liftcycle.
```

Comment:
Remove this function as it turned out that we have the same logic implemented in the `shouldComponentUpdate`.